lib/foreman_ansible/register.rb

```diff
@@ -257,4 +259,16 @@ def ansible_variables_with_overrides
                         :partial => 'foreman/smart_proxies/update_smart_proxy',
                         :onlyif => ->(proxy, view) { view.can_update_proxy?(proxy) }
   end
+  extend_page('templates/_form') do |context|
+    context.add_pagelet :tab_headers,
+                        :name => _('Ansible Callback'),
+                        :partial => 'job_templates/job_template_callback_tab_headers',
+                        :onlyif => ->(subject, view) { view.correct_template? }
```

Comment:
@adamruzicka is there a way to implement this method call in one line? (without having to create the `JobTemplatesHelper`?)

Reply:
For some reason I am getting that `subject` is nil

Reply:
Ah, interesting. It works for `:tab_content`. Most likely because you are not providing the subject for the headers in here https://github.com/theforeman/foreman/pull/9503/files#diff-652d7608bcc9387e191b8c1a53a8cf45bb75cbf21e776863d7b14ca206d9774fR15

Reply:
I found out why. I didn't pass it in foreman core, when calling the `render_pagelets_for(:tab_headers)`.